pkgs/applications/editors/kile/frameworks.nix

```diff
@@ -60,8 +60,9 @@ let
       };
     };
 in
-kdeWrapper unwrapped
+kdeWrapper
 {
+  unwrapped = [ unwrapped ];
```

Comment:
Please make `kdeWrapper` able to accept a list of derivations *or* a single derivation as `unwrapped` so that we may simply do `inherit unwrapped;` here.